### ISSUE TITLE
Show tooltips on elements that have keyboard focus

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#52016]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS2.1.1,MAS1.1.1

GitHubTags: #A11yMAS; #A11yTCS; #Win32; #DesktopApp #A11y_VSCodeEditor #WCAG2.1.1 #WCAG1.1.1 

Repro Steps:
1. Launch VS Code.
2. Go to View menu and Select Problems.
3. Tab till collapse button beside Filter textbox in Problems Tab.

Actual:
Tooltip of buttons(Collapse,Minimize Panel Size, Move to right, Close Panel) in the Problems tab are not available when we navigate using keyboard. Tooltips are only available on mouse hover.

Expected:
When keyboard focus goes on control and for that control if tooltip is present then on keyboard focus tooltip should also appear.

User Impact:
Keyboard only users wont get the information which are conveyed through tooltip.

Attachment for Reference: 
[A11y VSCode Problems NVDA Tooltipnotaccessible.pptx](https://github.com/Microsoft/vscode/files/2105689/A11y.VSCode.Problems.NVDA.Tooltipnotaccessible.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
Assigning to @bpasero as this might impact all workbench actions

**Comment by bpasero** [comment]
@AccessibilityTestingTeam-TCS I cannot reproduce, when I tab over the buttons I get the following: 

![image](https://user-images.githubusercontent.com/900690/41534695-b3618644-72ff-11e8-9fdc-fb31d409756b.png)

**Comment by bpasero** [comment]
We do not control that, it is the native tooltip.

**Comment by isidorn** [comment]
@bpasero thanks

**Comment by isidorn** [comment]
@bpasero if we introduced some setting for this. Is there some way for us to always trigger the hover as the focus moves.

**Comment by bpasero** [comment]
@isidorn to my knowledge you cannot programmatically trigger the native tooltip in the browser

**Comment by nirmalsahoo** [comment]
Same issue is observed in below vs code version.
Version: 1.44.2 (user setup)
Commit: ff915844119ce9485abfe8aa9076ec76b5300ddd
Date: 2020-04-16T16:36:23.138Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Comment by MayuriNaik** [comment]
@isidorn Same issue is observed in below vs code version. It is reproduced for all the controls for which the tooltips are visible by mouse but not by Keyboard.
Version: 1.41.0-insider
Commit: 9579eda04fdb3a9bba2750f15193e5fafe16b959
Date: 2019-12-10T21:42:59.146Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: MAC_Darwin x64 19.5.0

**Comment by isidorn** [comment]
From the user perspective we already read the aria label of the element once the user navigates on it. Aria label is usually the same as what is hovered or better, so this issue is covered for blind users.
The only users this makes sense for imho are users who can not use the mouse and are not using a screen reader. Thus they navigate using the keyboard to an action and they would like to know more details about that action. So it would be nice to have an explicit command to trigger this native hover for those users. However web browsers do not support this as far as I know and it is not possible to trigger a native hover.

We can maybe look into this once MS Teams starts supporting this. 
The only solution I see is to build a custom hover which I would not do for a feature which none of the users requested.

**Comment by Tanushagottipati** [comment]
GitHub Tags: #A11ySev2

**Comment by isidorn** [comment]
As per my comment [above](https://github.com/Microsoft/vscode/issues/52016#issuecomment-641096443) and because none of our users ever requested this I am closing this as designed. We will not change this.

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/132344
